src/main.rs

```diff
@@ -97,6 +103,10 @@ fn main() -> Result<()> {
     app.start_background_thread();
     let res = run_draw_loop(&mut terminal, app);
 
+    if fd < 0 {
+        fs::write(BPF_STATS_ENABLED, b"0")?;
```

Comment:
If the app panics, this line is not guaranteed to run and stats will remain enabled. This is not an issue with the `_owned_fd` approach since it will auto close when the app exits. We need another approach that guarantees we disable stats if the app panics.